src/code/z_vismono.c

```diff
@@ -1,13 +1,15 @@
 /**
+ * @file z_vismono.c
+ *
  * Color frame buffer effect to desaturate the colors.
```

Comment:
maybe we can expand this comment just a little bit, "the colors" sounds abrupt and incomplete

Reply:
(i know this wasnt a part of the original set of changes, but yeah)